.docker/office/Dockerfile

```diff
@@ -1,18 +1,33 @@
-FROM openjdk:25-bookworm
-LABEL maintainer="ITK Development <itkdev@mkb.aarhus.dk>"
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y install \
-  libreoffice-common \
-  unoconv \
-  hyphen-af hyphen-en-us \
-  default-jre \
-  libreoffice-java-common \
-  fonts-dejavu fonts-dejavu-core fonts-dejavu-extra \
-  fonts-droid-fallback fonts-dustin fonts-f500 fonts-fanwood fonts-freefont-ttf fonts-liberation \
-  fonts-lmodern fonts-lyx fonts-sil-gentium fonts-texgyre fonts-tlwg-purisa fonts-opensymbol && \
-  rm -rf /var/lib/apt/lists/*
-
-EXPOSE 8100
-
-ENTRYPOINT ["/usr/bin/libreoffice"]
-CMD ["--headless", "--nologo", "--nofirststartwizard", "--accept=\"socket,host=0.0.0.0,port=8100;urp\""]
+# pull pyhton
+FROM python:3.11-bookworm
+
+# Update PIP
+RUN apt update && \
+    pip install --no-cache-dir --upgrade pip
+
+WORKDIR /app
+
+# Install Libreoffice
+RUN apt install -y libreoffice-common
+RUN apt install -y libreoffice-writer
```

Comment:
The more run commands you can stack into one rune command the small the image get.

Reply:
Also adding " rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*" to clean up the image and final size.

Reply:
```
RUN DEBIAN_FRONTEND=noninteractive \
    apt-get install -qy \
    libreoffice-common \
    libreoffice-writer
&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```